cms/pytest.ini

```diff
@@ -12,6 +12,11 @@ filterwarnings =
     default
     ignore:No request passed to the backend, unable to rate-limit:UserWarning
     ignore::xblock.exceptions.FieldDataDeprecationWarning
+    ignore:.*You can remove default_app_config.*:PendingDeprecationWarning
+    ignore:.*You can remove default_app_config.*:DeprecationWarning  # Class depends on Django version
+    ignore:.*Instead access HTTPResponse.headers directly.*:DeprecationWarning  # Warning from elasticsearch client
+    ignore:.*Using or importing the ABCs from 'collections' instead of from 'collections.abc' is.*:DeprecationWarning  # Warning from libsass
+    ignore:.*Deprecated call to `pkg_resources.declare_namespace.*:DeprecationWarning  # XBlock warning https://github.com/openedx/XBlock/issues/641
```

Comment:
Are you sure none of these warnings comes directly from `edx-platform`? We want to catch (and fix) things that will be removed in future releases of its dependencies.

Could we explicitly define the packages these warnings come from? Example: `ignore::xblock.exceptions.FieldDataDeprecationWarning` specifies that the warning comes from the `XBlock` package.

Reply:
> Are you sure none of these warnings comes directly from edx-platform? We want to catch (and fix) things that will be removed in future releases of its dependencies.

Yeah i tried to make sure of that, but I can review again.

> Could we explicitly define the packages these warnings come from? Example: ignore::xblock.exceptions.FieldDataDeprecationWarning specifies that the warning comes from the XBlock package.

that's a really good idea, will do that